api/apps/articles/model.coffee

```diff
@@ -40,6 +40,7 @@ schema = (->
   thumbnail_image: @string().allow('', null)
   tags: @array().items(@string())
   title: @string().allow('', null)
+  keywords: @string().allow('', null)
```

Comment:
Going for array on the data end, this would just need to be `@array().items(@string())` like `tags` above.